packages/sp/src/webs.ts

```diff
@@ -111,7 +111,7 @@ export class Web extends SharePointQueryableShareableWeb {
      */
     public getParentWeb(): Promise<{ data: any; web: Web }> {
         return this.select("ParentWeb/Id").expand("ParentWeb").get()
-            .then(({ ParentWeb }) => new Site(this.toUrlAndQuery().split("/_api")[0]).openWebById(ParentWeb.Id));
+            .then(({ ParentWeb }) => ParentWeb ? new Site(this.parentUrl).openWebById(ParentWeb.Id) : null);
```

Comment:
Why not here just get the ParentWeb/Url and return a new web that way? I am sure I wrote the old code, but looking at it I think we can come up with a better way.

Reply:
Here a previous discussion about the implementation #69

Reply:
Oh right, ParentWeb isn't a web instance but WebInformation instance. Got it.